packages/viewer/src/index.html

```diff
@@ -35,8 +35,38 @@
       async function main() {
         const viewer = document.querySelector('#viewer');
 
-        // viewer.token = 'my-token'
-        // await viewer.load('urn:vertexvis:scene:cd78e291-760e-4bf5-8b77-c9c981847d0e')
+        viewer.token = 'OFQkyrmGbOVSs560k4doTfZQ2b_LRckgNYBRMW3tRt8.fDKosxo468Wi_z3ig75Q5cOsXIAVP5omJWGC-3znwfs'
+        await viewer.load('urn:vertexvis:scene:51ba5e31-82c6-4f64-bcf5-349ec5802425');
+
+        viewer.addEventListener('tap', async function(event) {
+          console.log('tapping: ', event);
+          const { position } = event.detail;
+          const scene = await viewer.scene();
+          const raycaster = await scene.raycaster();
+
+          // const test = scene.material("1234").material("1111").execute();
+
+          const abc = scene
+            .itemOperation()
+            .where(q => q.itemId('abc').or().q.suppliedId('123'))
```

Comment:
`.or().q` is `.q` needed?

Reply:
yeah, it should be. This wasn't working yet.

Reply:
.q() should not be needed and does not work. Addressed.